actions.go

```diff
@@ -56,7 +56,7 @@ type CustomActionsTriggerDefinitionsConnection struct {
 
 func (client *Client) CreateWebhookAction(input CustomActionsWebhookActionCreateInput) (*CustomActionsExternalAction, error) {
 	var m struct {
-		Payload struct {
+		Payload struct { // TODO: fix this
```

Comment:
All the "TODO" are there for future self's when we come back around.  Unfortunately switching these inline structs to the generated payload stucts requires test changes that i'm not willing to take on right now.  So i've just noted all the places where we need to fix for the future.